### DISCUSSION TITLE
contribution histogram now requires scrolling?

### DISCUSSION THREAD

**Discussion by cwreynolds** [community/community#72880] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

This may not be the right place, and it could easily be user error, but:

In the last month or so, something seems to have changed in the (html/css?) layout of my profile page. The “contribution histogram” no longer fits in its layout box, so I need to scroll it to the left to see recent entries. Initial display:

> <img width="911" alt="image" src="https://github.com/community/community/assets/1909341/7dac36f1-b11d-40b3-8e3e-e5ecadd9ba32">

After scrolling:

> <img width="911" alt="image" src="https://github.com/community/community/assets/1909341/29c75769-1290-43e2-8b93-b9cfa06176d8">

Wondering if this was just me, I checked a friend's profile and saw that his “histogram” was scaled to fit. But he had the Activity Overview display turned on. When I turn mine on, my histogram is then scaled to fit:

> <img width="911" alt="image" src="https://github.com/community/community/assets/1909341/77194f39-92a9-4671-acb8-e7aff43d163a">

Is this a bug? If so does anyone know where I should report it?

**Comment by cotes2020** [comment, marked as answer]
It's basically considered a bug, and when you log out of your GitHub account, the scrollbar will go away and the contribution history graph will return to normal.

Maybe you can submit a  ticket to [GitHub Support](https://support.github.com/contact/bug-report) and it will be passed on to the GitHub development team.

**Comment by cwreynolds** [comment]
OH! If I log out, but _not_ back in, then it is OK. Now I see. So it is a layout problem which is “only” seen by everyone logged into GitHub.

Thanks and I filed a [support ticket](https://support.github.com/ticket/personal/0/2406050)

**Comment by ricardonunesdev** [comment]
+1

Same issue here, when logged in the years selector is beside the contribution graph, which now has a horizontal scrollbar. When logged out it's fine.

**Comment by cwreynolds** [comment]
Wow!!
It is fixed!
Very quick turn-around. Thanks for help here and on https://support.github.com/ticket/personal/0/2406050

**Comment by cotes2020** [comment]
@cwreynolds  Tickets are only readable by the submitter :)

Also, I don't think this fix is perfect though. The layout is still different for logged in and non-logged in states. 

|Log in| Log out|
|:--|:---|
| ![login](https://github.com/community/community/assets/11371340/0485ae9f-9bd8-4a91-9bb6-05b1549303d4) | ![logout](https://github.com/community/community/assets/11371340/a224db38-22d1-46da-ba9d-3cd19d58120b)|

Note the position of the year's timeline: one is on the right, while the other is at the bottom.
